CHANGELOG.md

```diff
@@ -5,6 +5,24 @@
 * **Potentially breaking bug fix**: Selectors with unmatched brackets now always
   produce a parser error. Previously, some edge cases like `[foo#{"]:is(bar"}) {a:
   b}` would compile without error, but this was an unintentional bug.
+## 1.83.5-dev
```

Comment:
Get rid of this header.

Reply:
@nex3 I removed that. I also removed an extra period at the end of the next point that I noticed. (see https://github.com/sass/dart-sass/pull/2467/commits/7c6d741430ffcd60a0b6cbcd61acf6ff842b41e9)